lib/src/assets/img/logo-nexus-generic.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="72px" height="72px" viewBox="0 0 72 72" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Group 7 Copy</title>
+    <g id="Navigation" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Nexus-Headers" transform="translate(-81.000000, -579.000000)">
+            <g id="Group-7-Copy" transform="translate(81.000000, 579.000000)">
+                <rect id="Rectangle" fill="#949494" x="0" y="0" width="72" height="72"></rect>
```

Comment:
This is the grey background?  Wouldn't we want the SVG to only be the hexagon shape and we'd do the background color in HTML/CSS?

Reply:
> This is the grey background? Wouldn't we want the SVG to only be the hexagon shape and we'd do the background color in HTML/CSS?

SVG extensively cleaned up here: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/e6ac7a8ea2c55cc21584f554f7ae76b798036799